packages/playground/src/types.ts

```diff
@@ -537,3 +543,68 @@ export interface FunctionSchedulesFilter {
   createdTime?: Date;
   cronExpression?: string;
 }
+
+/* ************************
+   Annotations API types
```

Comment:
can you put this in a types.ts file inside the annotations api folder?
Then just do the following here:
```ts
export * from './api/annotations/types.gen';
```

Reply:
Done